### DISCUSSION TITLE
ignore a specific package, in a specific path

### DISCUSSION THREAD

**Discussion by Wassap124** [renovatebot/renovate#26896] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

I have a mono-repo `ts` project with several microservices, each having its own `pacakge.json`.
there's a specific package i want `renovate` to ignore, but only in a specific micro-service.

is that possible? something that'd be like so: 
```
{
  "packageRules": [
    {
      "matchPaths": ["services/myservice"],
      "packageNames": ["some-package"],
      "enabled": false
    }
  ]
}
```

that `matchPaths` option doesn't exist and i couldn't find anything similar in the docs.
Thanks!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
https://docs.renovatebot.com/configuration-options/#matchfilenames
https://docs.renovatebot.com/configuration-options/#matchpackagenames

**Reply by Wassap124** [reply]
Thanks ❤️
